src/test/java/seedu/address/model/person/passenger/PriceContainsKeywordsPredicateTest.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.person.passenger;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.PassengerBuilder;
+
+public class PriceContainsKeywordsPredicateTest {
+
+    @Test
+    public void equals() {
+        Double firstPredicatePrice = 2.34;
+        Double secondPredicatePrice = 10.50;
```

Comment:
should these be magic literals? could we have a sample price constant from the test util classes here?